core/src/main/java/org/infinispan/notifications/cachemanagerlistener/CacheManagerNotifierImpl.java

```diff
@@ -133,7 +135,10 @@ private void invokeListener(ListenerInvocation listener, EventImpl e) {
       } catch (Exception x) {
          // Only cache entry-related listeners should be able to throw an exception to veto the operation.
          // Just log the exception thrown by the invoker, it should contain all the relevant information.
-         log.error(x);
```

Comment:
-1 to print the stack trace only if trace is enabled. If the exception is harmless and shouldn't be logged, the listener can catch it itself.